### PR TITLE
plat-totalcompute: Introduce TC2

### PR DESCRIPTION
**Pull request by rupsin01**
Added TC2 platform support

Signed-off-by: Rupinderjit Singh <rupinderjit.singh@arm.com>

### CODE REVIEW

File: core/arch/arm/plat-totalcompute/platform_config.h [hunk 1]
```diff
@@ -11,7 +11,9 @@
 /* Make stacks aligned to data cache line length */
 #define STACK_ALIGNMENT		64
 
-#if defined(PLATFORM_FLAVOR_tc0) || defined(PLATFORM_FLAVOR_tc1)
+#if (defined(PLATFORM_FLAVOR_tc0) ||	\
+defined(PLATFORM_FLAVOR_tc1) ||	\
+defined(PLATFORM_FLAVOR_tc2))
```
Comment (etienne-lms) [inline core/arch/arm/plat-totalcompute/platform_config.h:16]: fix indentation:
```c
#if (defined(PLATFORM_FLAVOR_tc0) ||	\
     defined(PLATFORM_FLAVOR_tc1) ||	\
     defined(PLATFORM_FLAVOR_tc2))

Comment (rupsin01) [reply core/arch/arm/plat-totalcompute/platform_config.h]: >> checkpatch 8f64138f5480768c421b461a2053e146a940fe71
8f64138f plat-totalcompute: Introduce TC2
WARNING: please, no spaces at the start of a line
#43: FILE: core/arch/arm/plat-totalcompute/platform_config.h:15:
+     defined(PLATFORM_FLAVOR_tc1) ||^I\$

WARNING: please, no spaces at the start of a line
#44: FILE: core/arch/arm/plat-totalcompute/platform_config.h:16:
+     defined(PLATFORM_FLAVOR_tc2))$

total: 0 errors, 2 warnings, 0 checks, 23 lines checked
##[error]Bash exited with code '1'.

Earlier I was doing the indentation that way and got these errors.

Comment (jforissier) [reply core/arch/arm/plat-totalcompute/platform_config.h]: Use one tab

**Comment by rupsin01** [comment]
Space issue:
failed to register layer: ApplyLayer exit status 1 stdout:  stderr: write /root/optee_repo_qemu_v8/u-boot/drivers/ddr/fsl/ddr4_dimm_params.c: **no space left on device**

**Review by jforissier — APPROVED** [review]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jforissier** [comment]
> Space issue: failed to register layer: ApplyLayer exit status 1 stdout: stderr: write /root/optee_repo_qemu_v8/u-boot/drivers/ddr/fsl/ddr4_dimm_params.c: **no space left on device**

The Docker image hasn't changed recently so it must be something related to the Azure test infrastructure. I will investigate on Monday when I'm back from vacation.

**Review by etienne-lms — COMMENTED** [review]
`Acked-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Comment by rupsin01** [comment]
@jforissier / @etienne-lms Good Morning! Please help review and merge this patch.

**Comment by jforissier** [comment]
Hi @rupsin01, please add the `Acked-by:` tags and I will merge this. Thanks!

**Comment by rupsin01** [comment]
> 

Added the Acked-by tag. Thanks

**Comment by rupsin01** [comment]
Thanks @jforissier for merging the patch.

**Pull request merged**